client/ayon_core/pipeline/create/context.py

```diff
@@ -1468,6 +1467,267 @@ def execute_autocreators(self):
         if failed_info:
             raise CreatorsCreateFailed(failed_info)
 
+    def get_folder_entities(self, folder_paths: Iterable[str]):
+        """Get folder entities by paths.
+
+        Args:
+            folder_paths (Iterable[str]): Folder paths.
+
+        Returns:
+            Dict[str, Optional[Dict[str, Any]]]: Folder entities by path.
+
+        """
+        output = {
+            folder_path: None
+            for folder_path in folder_paths
+            if folder_path is not None
```

Comment:
According to the input type hint this should never be 'allowed' as value in the Iterable?